src/services/api/private-get-rea/index.ts

```diff
@@ -0,0 +1,42 @@
+import PrivateApiClient from "./privateApiClient";
+import { API_URL, ACCOUNT_URL } from "./config";
+import { RequestClient, HttpClientOptions, IHttpClient } from "@companieshouse/api-sdk-node/dist/http";
+import Resource from "@companieshouse/api-sdk-node/dist/services/resource";
+
+/**
+ * Creates a new API Client.
+ *
+ * @param apiKey the api key to use for authentication
+ * @param oauthToken a user's oauth token that can be used for authentication
+ * @param baseUrl the api base url
+ */
+export const createPrivateApiClient = (apiKey?: string, oauthToken?: string, baseUrl: string = API_URL, baseAccountUrl: string = ACCOUNT_URL): PrivateApiClient => {
+  if (apiKey && oauthToken) {
+    throw new Error("You cannot set both api key and oauth token to create a client. Please use one or the other");
```

Comment:
Rather than directly inline the error message here, can we add to `constants/app.const.ts` and reference that constant here for consistency?

Reply:
Done